lib/orderbook/OrderBookRepository.ts

```diff
@@ -50,6 +50,14 @@ class OrderbookRepository {
   public addTrade = (trade: db.TradeFactory) => {
     return this.models.Trade.create(trade);
   }
+
+  public getTradesLimit = (limit?: number): Bluebird<db.TradeInstance[]> => {
```

Comment:
Can we just call this `getTrades`? I think that makes more sense since the limit is optional.